cart/interfaces/graphql/mutationresolvers.go

```diff
@@ -235,11 +220,9 @@ func (r *CommerceCartMutationResolver) CommerceCartUpdateDeliveryShippingOptions
 		if err != nil {
 			return nil, err
 		}
-		formResult.Processed = true
-		result = append(result, &formResult)
 	}
 
-	return result, nil
+	return &dto.UpdateShippingOptionsResult{Processed: true}, nil
```

Comment:
Do we need the `Processed` field if it's always true?

Reply:
Thought about just resolving with true and decided to be keep the return type with one and only process field so we can extend it in the future.